objects/client.c

```diff
@@ -805,9 +805,17 @@
  * @function set_newindex_miss_handler
  */
 
+typedef enum {
+    CLIENT_MAXIMIZED_NONE = 0 << 0,
+    CLIENT_MAXIMIZED_V    = 1 << 0,
+    CLIENT_MAXIMIZED_H    = 1 << 1,
+    CLIENT_MAXIMIZED_BOTH = 1 << 2, /* V|H == BOTH, but ~(V|H) != ~(BOTH)... */
```

Comment:
I was referring to the concepts. When you maximize both H and V, then you are *in effect* maximized. But when you undo the operation, you are left in a potentially inconsistent state.

I agree the comment is unhelpful and misleading. I should copy/paste the [discussion comment](https://github.com/awesomeWM/awesome/pull/1453#discussion_r97938954) into the code and remove this comment.